recipe/meta.yaml

```diff
@@ -47,75 +57,125 @@ outputs:
         - LICENSE.txt
       summary: A meta-package to select Arrow build variant
 
-  - name: arrow-cpp
+  # compat output for old mutex-package naming
+  - name: arrow-cpp-proc
+    version: {{ build_ext_version }}
+    build:
+      number: {{ proc_build_number }}
+      string: {{ build_ext }}
+      ignore_run_exports_from:
+        - openssl
+    requirements:
+      host:
+        # ensure we split the CI jobs per openssl version
+        - openssl
+      run:
+        - apache-arrow-proc ={{ build_ext_version }}={{ build_ext }}
+    test:
+      commands:
+        - exit 0
+
+  - name: libarrow
     script: build-arrow.sh  # [not win]
     script: bld-arrow.bat   # [win]
     version: {{ version }}
     build:
-      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
       run_exports:
-        - {{ pin_subpackage("arrow-cpp", max_pin="x.x.x") }}
+        - {{ pin_subpackage("libarrow", max_pin="x.x.x") }}
       ignore_run_exports:
         - cudatoolkit
       track_features: {{ "[arrow-cuda]" if cuda_enabled else "" }}
+      missing_dso_whitelist:
+        - "*/libcuda.so.*"    # [linux]
+        - "*/nvcuda.dll"      # [win]
     requirements:
       build:
-        - python                                 # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - cython                                 # [build_platform != target_platform]
-        - numpy                                  # [build_platform != target_platform]
         - clangdev {{ llvm_version }}            # [osx and arm64]
         - llvmdev {{ llvm_version }}             # [osx and arm64]
         - gnuconfig                              # [osx and arm64]
         - libgrpc
         - libprotobuf
-        # aws-sdk-cpp 1.8.* doesn't work with a newer CMake
-        - cmake <3.22
+        - cmake
         - autoconf  # [unix]
         - ninja
         - make  # [unix]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler("cuda") }}  # [cuda_compiler_version != "None"]
       host:
-        # https://issues.apache.org/jira/browse/ARROW-15141
-        - aws-sdk-cpp 1.8.186
-        # abseil is only here to help conda pick the right constraints for pyarrow, see
-        # https://github.com/conda-forge/arrow-cpp-feedstock/pull/815#issuecomment-1216713245
-        - libabseil
+        - clangdev {{ llvm_version }}
+        - llvmdev {{ llvm_version }}
+        - aws-sdk-cpp
         - boost-cpp >=1.70
         - brotli
         - bzip2
         - c-ares
         - gflags
         - glog
         - google-cloud-cpp
+        - libabseil
```

Comment:
Does `libabseil` need to be explicitly included here? (above there was a comment "abseil is only here to help conda pick the right constraints for pyarrow .." that is removed now)

Reply:
Yes, see https://github.com/conda-forge/arrow-cpp-feedstock/pull/875/commits/368b437c42f9e2fdf531732dbbb29968667b332d

Reply:
It might be good to keep some comment about that then?

Reply:
Hm, I'm not sure. It's just a regular dependency at this point, and for those that want the details, they just need to look at the blame (I'm planning not to squash the commits, because they're intentionally granularized into sensible/separate chunks)

Reply:
> It's just a regular dependency at this point,

Ah, yes, forgot that we now actually use it directly in the gcfs code